roles/common/files/download_start.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 sudo systemctl stop solana-validator.service
-python3 /mnt/snapshot-finder.py --snapshot_path /mnt/solana-snapshots
+python3 /home/ubuntu/snapshot-finder.py --snapshot_path /solana/acconts
```

Comment:
i don't think we want to store snapshots in accounts folder (also typo in accounts).  typically you either store them in a separate snapshots folder (which i'm starting to like) or you put them in the ledger dir

Reply:
yeah, you have it set to ledger dir below, so i don't think this would work